help(Continuous-integration): re-organize the section (#150)
bump-sh/docs: src/_help/continuous-integration/github-actions.md

## Generate your API documentation & changelog

> Bump.sh helps you build a branded single source of truth, cataloging all your APIs. We’ve created the reference point for teams consuming and building APIs, no matter which technology they rely on.

Reviewer: Not sure about that part. I don't think that we should promote Bump.sh at this step.

Bump.sh keeps your API docs always synchronized with your codebase. With this [Github Action](https://github.com/actions) your API reference is automatically generated - with changelog and diff - on [Bump.sh](https://bump.sh) from any [OpenAPI](https://github.com/OAI/OpenAPI-Specification) or [AsyncAPI](https://github.com/asyncapi/asyncapi) file.

## Usage

Start with creating a documentation on [Bump.sh](https://bump.sh).

Once you've got an API Documentation set up, go to Settings > CI Deployment, copy the access token, then add it to GitHub Settings > Secrets > Actions as a new environment variable called `BUMP_TOKEN`.

Then you can pick from one of the three following API workflow files.

- **Recommended:** [Deploy Documentation & Diff on Pull Requests](#deploy-documentation--diff-on-pull-requests)
- [Deploy Documentation only](#deploy-documentation-only)
- [Diff on Pull Requests only](#diff-on-pull-requests-only)

### Deploy Documentation & Diff on Pull Requests

> This is the most common worklow that we [recommend using](https://help.bump.sh/continuous-integration#integrate-with-your-ci), which will create two steps in your automation flow: a validation & diff step on code reviews, followed by a deployment step on merged changes.

Reviewer: ```suggestion
This is the [recommend workflow](https://help.bump.sh/continuous-integration#integrate-with-your-ci), which will create two steps in your automation flow: a validation & diff step on code reviews, followed by a deployment step on merged changes.
```

          doc: <BUMP_DOC_ID>
          token: ${{secrets.BUMP_TOKEN}}
          file: doc/api-documentation.yml
```

> make sure to change your destination branch name (`main` in the example above), replace `<BUMP_DOC_ID>` with your Bump.sh documentation slug or id, and point `file:` to your local API description document (`doc/api-documentation.yml`).
{: .warning}

> ### Diff on Pull Requests only

Reviewer: ```suggestion
### Diff on pull requests only
```

> If you only want to have API diff posted on pull requests use this workflow:

Reviewer: ```suggestion
If you only want to have API diff posted on pull requests, use this workflow:
```

## Code of Conduct

> Everyone interacting in the Bump `github-action` project’s codebases, issue trackers, chat rooms and mailing lists is expected to follow the [code of conduct](https://github.com/bump-sh/github-action/blob/master/CODE_OF_CONDUCT.md).

Reviewer: ```suggestion
Everyone interacting in the Bump.sh `github-action` project’s codebases, issue trackers, chat rooms and mailing lists is expected to follow the [code of conduct](https://github.com/bump-sh/github-action/blob/master/CODE_OF_CONDUCT.md).
```
